src/Components/TheoryPage.js

```diff
@@ -0,0 +1,989 @@
+import React, { useState } from 'react';
+import { View, StyleSheet, Text, Dimensions, FlatList, SectionList, Modal, TouchableHighlight } from 'react-native';
+import { createMaterialBottomTabNavigator } from '@react-navigation/material-bottom-tabs';
+import { Ionicons, MaterialCommunityIcons } from '@expo/vector-icons';
+
+import VimeoVideo from './VimeoVideo';
+
+import theme from '../styles/theme.style';
+
+const Tab = createMaterialBottomTabNavigator();
+const screenDimension = Dimensions.get('window');
+
+const Dictionary = () => {
+  const [modalVisible, setModalVisible] = useState(false);
+  const [selectedItem, setSelectedItem] = useState({
+    item: {
+      id: 0,
+      text: '',
+      definition: '',
+    },
+  });
+
+  const sections = [
```

Comment:
I feel like this should be extracted in another file, probably within the fixtures folder, and using the same structure as the drills so that we can then translate everything